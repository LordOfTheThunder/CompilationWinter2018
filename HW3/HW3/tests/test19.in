int potato() {
	int t = 13;
}
void tomato() {
	int x = potato();
}

void main() { print("Max is cool"); }
