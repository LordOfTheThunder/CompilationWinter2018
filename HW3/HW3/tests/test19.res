---end scope---
t INT 0
---end scope---
x INT 0
---end scope---
print (STRING)->VOID 1
printi (INT)->VOID 2
potato ()->INT 3
tomato ()->VOID 4
