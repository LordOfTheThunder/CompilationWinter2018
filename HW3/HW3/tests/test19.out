---end scope---
t INT 1
---end scope---
x INT 1
---end scope---
print (STRING)->VOID 1
printi (INT)->VOID 1
potato ()->INT 1
tomato ()->VOID 1
